core-plugins/src/e2e-test/java/io/cdap/plugin/common/stepsdesign/CorePlugin.java

```diff
@@ -126,4 +131,24 @@ public void validateOutputOrcFileGeneratedByFileSinkPluginIsEqualToExpectedOutpu
         PluginPropertyUtils.pluginProp(expectedOutputFilePath));
     }
   }
+
+  @Then("Validate that file gets deleted or moved from the gcs bucket")
+  public static boolean validateThatFileGetsDeletedFromTheGcsBucket() {
```

Comment:
Please test delete and moved separately. For moved you need to check the file was successfully transferred to the target location

Reply:
added separately.